qm-clone: add page (#9038)
tldr-pages/tldr: pages/linux/qm-clone.md

# qm clone

> Create a copy of virtual machine on QEMU/KVM Virtual Machine Manager.
> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.

- Copy / Clone a virtual machine:

`qm copy {{100}} {{102}}`

- Set a name for the cloned VM:

`qm copy {{100}} {{102}} --name {{name}}`

> - Set a description for the cloned VM:

Reviewer: ```suggestion
- Copy a virtual machine using a specific descriptionn:
```

`qm copy {{100}} {{102}} --description {{description}}`

> - Create a full clone of the source VM:

Reviewer: ```suggestion
- Copy a virtual machine creating a full copy of all disks:
```

`qm copy {{100}} {{102}} --full`

> - Specify file format for the cloned VM (requires full clone):

Reviewer: ```suggestion
- Copy a virtual machine using a specific format for file storage (requires a --full clone):
```

> `qm copy {{100}} {{102}} --full --format {{qcow2 | raw | vmdk}}`

Reviewer: ```suggestion
`qm copy {{100}} {{102}} --full --format {{qcow2|raw|vmdk}}`
```

> - Specify pool for the cloned VM:

Reviewer: ```suggestion
- Copy a virtual machine then add it to a specific pool:
```

> - Copy a virtual machine using a specific format for file storage (requires a --full clone):

Reviewer: ```suggestion
- Copy a virtual machine using a specific format for file storage (requires `--full`):
```
